notebook/static/services/sessions/session.js

```diff
@@ -38,7 +38,7 @@ define([
 
         this.base_url = options.base_url;
         this.ws_url = options.ws_url;
-        this.session_service_url = utils.url_join_encode(this.base_url, 'api/sessions');
+        this.session_service_url = this.base_url + 'api/sessions';
```

Comment:
to ensure slashes are correct, you can use `utils.url_join` instead of `url_join_encode`.